src/main/java/com/microsoft/aad/msal4j/AuthenticationErrorCode.java

```diff
@@ -9,19 +9,21 @@
 public class AuthenticationErrorCode {
 
     /**
-     * In the context of device code user has not yet authenticated via browser
+     * In the context of device code user has not yet authenticated via browser. For more details,
+     * see https://aka.ms/msal4j-device-code
```

Comment:
Can we update this aka.ms link to https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-desktop-acquire-token?tabs=java#command-line-tool-without-a-web-browser